### PR TITLE
Remove slash at the end of Swagger spec URL

### PR DESCRIPTION
**Pull request by oomichi**
If containing slash(/) at the end of URL at Swagger spec, current
apicoverage doesn't consider the corresponding APIs not-tested. For
example, "GET /api/v1" API is tested actually, but apicoverage doesn't
show the API as tested because of "GET /api/v1/" on Swagger spec.
This patch fixes it by removing slash from OpenAPI data.

fixes: #6181

### CODE REVIEW

File: experiment/coverage/apicoverage.go [hunk 1]
```diff
@@ -137,6 +138,8 @@ func getAPILog(restlog string) apiArray {
 	return parseAPILog(fp)
 }
 
+var reOpenapi = regexp.MustCompile(`({\S+?})`)
```
Comment (fejta) [inline experiment/coverage/apicoverage.go]: Move back?

Comment (oomichi) [reply experiment/coverage/apicoverage.go]: Yeah, the reOpenapi is used in getTestedAPIs() only.
So I just move it to here for the code readability.
Sorry for mixing it in this PR.

**Pull request merged**